src/queries/index.js

```diff
@@ -0,0 +1,10 @@
+export { default as archiveRemove } from './archiveRemove';
+export { default as archiveRemoveMany } from './archiveRemoveMany';
+export { default as archiveSplice } from './archiveSplice';
+export { default as pushInverse } from './pushInverse';
+export { default as pushOriginal } from './pushOriginal';
+export { default as put } from './put';
+export { default as putMany } from './putMany';
+export { default as remove } from './remove';
+export { default as spliceInverse } from './spliceInverse';
+export { default as spliceOriginal } from './spliceOriginal';
```

Comment:
Can we make these query names more clear? These are for internal use, so let's be as verbose as possible. I know what `put` does, for example, but a potential contributor would be super lost.